src/software/thunderscope/robot_diagnostics/robot_info.py

```diff
@@ -115,18 +116,32 @@ def __init__(
         # Stop primitive received indicator
         self.stop_primitive_label = QLabel()
         self.stop_primitive_label.setText("NA")
-        self.battery_layout.addWidget(self.stop_primitive_label)
+        self.stats_layout.addWidget(self.stop_primitive_label)
 
         # Primitive loss rate label
         self.primitive_loss_rate_label = common_widgets.ColorQLabel(
-            max_val=MAX_ACCEPTABLE_PACKET_LOSS_PERCENT
+            label_text="P%",
+            initial_value="NA",
+            max_val=MAX_ACCEPTABLE_PACKET_LOSS_PERCENT,
         )
-        self.battery_layout.addWidget(self.primitive_loss_rate_label)
 
-        self.battery_layout.addWidget(self.battery_progress_bar)
-        self.battery_layout.addWidget(self.battery_label)
+        # Primitive round-trip time label and queue
+        self.primitive_rtt_label = common_widgets.ColorQLabel(
+            label_text="RTT:",
+            initial_value="NA",
+            max_val=MAX_ACCEPTABLE_MILLISECOND_ROUND_TRIP_TIME,
+        )
```

Comment:
Could you add a `min_val` of 10 as well so the color only starts turning red if it's above 10ms?